build.githubactions.ps1

```diff
@@ -142,10 +148,20 @@ function Pack {
             dotnet pack $ProjectFile -c $Configuration --output $packageOutput --no-build --no-restore --verbosity normal -p:VersionPrefix=$version -p:NoWarn=NU5123
         }
     }
-    if ($NuspecFilePath -Like "*.nuspec" -and $PackageName -ne $null){
-       nuget pack $NuspecFilePath -OutputDirectory $packageOutput -Version $version -Properties configuration=$Configuration -Properties id=$PackageName -NoPackageAnalysis -NoDefaultExcludes
+    if ($NuspecFilePath -Like "*.nuspec" -and $null -ne $PackageName ){
+
+        if ($PackageName.Contains(".Extensions.")) { 
+            $RevisedPackageName = "$PackageName.$ExtensionVersion.Standard.$StandardVersion"
```

Comment:
this can also be removed if version and standard are included in the package name parameter

Reply:
Fixed it . Thanks